### PR TITLE
Build: Speed up building

### PR DESCRIPTION
**Pull request by WenheLI**
This PR speeds up building process

### CODE REVIEW

File: packages/cli/src/config.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const dependencies: string[] = [
 
 export const isLocal = process.env.NODE_ENV === 'local';
 export const daemonPackage = isLocal ? path.join(__dirname, '../../daemon') : '@pipcook/daemon';
-export const boardPackage = isLocal ? path.join(__dirname, '../../pipboard') : '@pipcook/pipboard';
+export const boardPackage = '@pipcook/pipboard';
```
Comment (rickycao-qy) [inline packages/cli/src/config.ts:10]: Should we also delete isLocal mode for daemon above?

Comment (yorkie) [reply packages/cli/src/config.ts:10]: Yea, we are able to remove local mode absolutely. /cc @FeelyChau

**Comment by yorkie** [comment]
@WenheLI Do we have any data to show how is the speed up?

**Comment by rickycao-qy** [comment]
master: 30.9s
this branch: 3.2s

File: packages/cli/src/config.ts (hunk 1 above)
Comment (FeelyChau) [reply packages/cli/src/config.ts:10]: It's done by #472

**Review by FeelyChau — APPROVED** [review]
LGTM.

**Pull request merged**